### PR TITLE
add support for build-args

### PR DESCRIPTION
**Pull request by nlf**

### CODE REVIEW

**Comment by bradrydzewski** [comment]
LGTM

File: DOCS.md [hunk 1]
```diff
@@ -59,6 +60,19 @@ publish:
...
+It's also possible to pass build arguments to docker:
+
+```yaml
+publish:
+  docker:
+    username: kevinbacon
+    password: pa55word
+    email: kevin.bacon@mail.com
+    repo: foo/bar
+		build_args:
```
Comment (tboerger) [inline DOCS.md]: Wrong indention

**Comment by tboerger** [comment]
After fixing the indent LGTM

**Comment by nlf** [comment]
fixed indenting and now it wants new LGTM comments

**Comment by nlf** [comment]
or not.. nevermind

**Pull request merged**